README.md

```diff
@@ -75,7 +77,7 @@ currently emitted:
 | Y041 | Y041 detects redundant numeric unions. For example, PEP 484 specifies that type checkers should treat `int` as an implicit subtype of `float`, so `int` is redundant in the union `int \| float`. In the same way, `int` is redundant in the union `int \| complex`, and `float` is redundant in the union `float \| complex`.
 | Y042 | Type alias names should use CamelCase rather than snake_case
 | Y043 | Do not use names ending in "T" for private type aliases. (The "T" suffix implies that an object is a `TypeVar`.)
-| Y044 | `from __future__ import annotations` has no effect in stub files, as forward references in stubs are enabled by default.
+| Y044 | `from __future__ import annotations` has no effect in stub files, since type checkers automatically treat stubs as having those semantics.
```

Comment:
This isn't strictly speaking true, since you can do crazy stuff in stubs that you can't do at runtime with `from __future__ import annotations` (e.g. subclassing a forward reference, etc.), but I haven't got a better suggestion :)